File: BEE 1114.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    while(1){
        int password;
        cin >> password;
        if (password == 2002){
            cout << "Acesso Permitido" << endl;
            break;
        }
        else{
            cout << "Senha Invalida" << endl;
        }
    }
    return 0;
}

File: BEE 1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(true){
    var password = parseInt(lines.shift());
    if (password == 2002){
        console.log("Acesso Permitido");
        break;
    }
    else{
        console.log("Senha Invalida");
    }
}